### PR TITLE
Add modulefunctionalities

### PR DESCRIPTION
**Pull request by BasitKorede**
# Add Es6 Sytax and Module 

- Create a new directory and initialize it with git.
- Use npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.
- Re-create your Awesome books app following the rules:

- Use ES6 syntax.
  - Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. 
 For this exercise - the more modules the better!
  - Keep all funcionalities of the app without errors.
- Refactor your methods and functions to arrow functions.
  - Use let and const in a correct way.
  - Display the current date below the navigation bar.
  - use luxon to update time and date.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+};
+formElement.addEventListener('submit', handleSubmition);
+
+// eslint-disable-next-line no-unused-vars
+const handleRemove = (currentId) => {
+  if (bookStore.remove(currentId)) {
+    const removeButton = document.getElementById(currentId);
+    removeButton.parentElement.parentElement.remove();
+  }
+};
```
Comment (dasileker) [inline js/index.js]: - [ ] The **HANDLE REMOVE** is not used and I can't delete items from the list.

kkk$

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @BasitKorede,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- No linters errors ✅.
- The PR is good ✅.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @BasitKorede, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Great job working on the required changes


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (BasitKorede) [reply js/index.js]: Thanks for your review 😊

**Pull request merged**